Microsoft.HealthVault.Fhir/Codes/HealthVault/HealthVaultVitalStatisticsCodes.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) Microsoft Corporation.  All rights reserved.
+// MIT License
+// Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the ""Software""), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+namespace Microsoft.HealthVault.Fhir.Codes.HealthVault
+{
+    using Hl7.Fhir.Model;
+
+    /// <summary>
+    /// This class is used to define the codeable values related to HealthVault Vital Statistics
+    /// </summary>
+    public static class HealthVaultVitalStatisticsCodes
+    {
+        public const string System = "http://healthvault.com/data-types/vital-statistics";
+
+        internal static Coding BodyWeight = new Coding()
+        {
+            Code = "Wgt",
```

Comment:
as discussed Vocabs are localized strings .. but I guess our intention is to use Vocab Micro service which supplies localized version ... this may not be applicable ..  

---
In reply to: [120748528](https://github.com/Microsoft/healthvault-fhir-library/pull/2#discussion_r120748528) [](ancestors = 120748528,120684279)